interactive/dashboard/flow_diagram.py

```diff
@@ -119,153 +94,1008 @@ def name_included_reactions(included_reactions, reactions_json):
         names_dict.update({reaction_index: name})
     return names_dict
 
+
+# check if element is in 'blocked' elements list
 def isBlocked(blocked, element_or_reaction):
```

Comment:
is this function still used? It looks like the code that did use it was deleted.